dwave/inspector/config.py

```diff
@@ -38,5 +38,9 @@ def host(self):
     def base_port(self):
         return int(self._env('DWAVE_INSPECTOR_BASE_PORT', 18000))
 
+    @property
```

Comment:
This could also be static, not sure if that's desirable or not

Reply:
static property?

Reply:
yes

Reply:
sorry, no. A static method. Which I suppose is why you did it this way. So ignore me.

Reply:
given they are not natively supported, and would have to muck with metaclass, not sure if that's worth it

Reply:
ah, didn't see your previous comment. github doesn't refresh sometimes